python/labbox_ephys/api/_workersession.py

```diff
@@ -99,18 +99,31 @@ def handle_message(self, msg):
         elif type0 == 'hitherCreateJob':
             functionName = msg['functionName']
             kwargs = msg['kwargs']
-            opts = msg['opts']
             client_job_id = msg['clientJobId']
-            if opts.get('newHitherJobMethod', False):
-                try:
-                    job = hi.run(functionName, **kwargs, labbox=self._labbox_context).wait()
-                except Exception as err:
-                    self._send_message({
-                        'type': 'hitherJobCreationError',
-                        'client_job_id': client_job_id,
-                        'error': str(err) + ' (new method)'
-                    })
-                    return
+            try:
+                outer_job = hi.run(functionName, **kwargs, labbox=self._labbox_context)
```

Comment:
This is the outer job... i.e. the hither job that creates and return a hither job to be queued.